deblur/workflow.py

```diff
@@ -46,16 +52,19 @@ def dereplicate_seqs(seqs_fp,
     seqs_fp : string
         filepath to FASTA sequence file
     output_fp : string
-        filepath to dereplicated FASTA file
+        file path to dereplicated sequences (FASTA format)
     min_size : integer
         discard sequences with an abundance value smaller
         than integer
+    dereplication_map: boolean
```

Comment:
can you change the name of this variable? Something like: `uc_output`
Also, you are missing the `, optional` part on the parameter definition, and describing the default in the string.